### ISSUE TITLE
After Using Next response from other controllers are empty

### ISSUE THREAD

**Issue by vikaskandari** [serhiisol/node-decorators#129]
This issue is happening in latest version of express decorator. 

When we use next function it works fine but other controllers send blank response after using next function example:
```typescript
import * as express from '@decorators/express/node_modules/express';
import { NextFunction } from '@decorators/express/node_modules/express';

import { attachControllers } from '@decorators/express';
import {
    Controller,
    Get,
    Next
} from '@decorators/express';



// Controller that will fire next function 
@Controller("*")
export class NextController {

    @Get("*")
    getA(@Next() next: NextFunction) {
        console.log("Next controller working");
        next();
    }

}

//This controller is working fine but the response is being sent empty even if we use res.send from arguments 
@Controller("/")
export class MainController {

    @Get("/")
    getB() {
        console.log("Controller is being called but res is being sent empty");
        return "This suppose to work after Next is being called";
    }
}


const app: express.Express = express();

const port = 3000;



attachControllers(app, [NextController, MainController]);

app.listen(port);

console.log('Server is running on http://localhost:'+port);


```

if you run the example in playground you will see its sending empty response but controller method is being called.

**Comment by vikaskandari** [comment]
Found the bug - We are actually sending the response even if controller is not returning anything which means if NextController we are sending the blank response and moving forward with MainController and when MainController is returning a value we are checking if headers are already sent or not thats why we are not getting any error .

**Comment by serhiisol** [comment]
Resolved in #136
